src/lib/button-manager.ts

```diff
@@ -86,6 +86,7 @@ export class ButtonManager {
           console.error(err);
         }
       }
+      return;
```

Comment:
it returns regardless on whether `await loadScript('https://pay.google.com/gp/p/js/pay.js');` succeeded or failed.
is it intentional? should the `return` be moved inside the `catch (err) {...}` block instead?

Reply:
nvm, I see it was address in a followup PR! sorry :)